wkhtmltopdf/utils.py

```diff
@@ -109,7 +109,7 @@ def wkhtmltopdf(pages, output=None, **kwargs):
 
     return check_output(ck_args, **ck_kwargs)
 
-def convert_to_pdf(filename, header_filename=None, footer_filename=None, cmd_options=None):
+def convert_to_pdf(filename_list, header_filename=None, footer_filename=None, cmd_options=None):
```

Comment:
I feel exactly the same as you, can't make my mind about what's better: keep the naming consistent or keep the api unbroken... @maxpeterson any thoughts?